### ISSUE TITLE
File/Dir mode regex's have false positives

### ISSUE THREAD

**Issue by timwsuqld** [warpnet/salt-lint#255]
**Describe the bug**
False positive for rule 207 and 208 for `mode` lines in other states

**To Reproduce**

1. State file for `network.managed`
```
bond0:
  network.managed:
    - type: bond
    - mode: 802.3ad
    - proto: static
    - ipaddr: 10.1.1.77
    - netmask: 255.255.255.0
    - gateway: 10.1.1.1
    - dns: 10.1.1.1
```
2. Run salt-lint against this file, it will fail with an error like:
```
[207] File modes should always be encapsulated in quotation marks
states/networking/server.sls:4
    - mode: 802.3ad

[208] File modes should always contain a leading zero
states/networking/server:4
    - mode: 802.3ad
```

**Expected behavior**
Only throw warnings for 207/208 when the mode line is part of a file/directory state.

**Desktop (please complete the following information):**

- OS: Ubuntu
- Version 20.04

**Comment by roaldnefs** [comment]
Thanks @timwsuqld for your bug report!
